TitaniumTask/app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,7 @@
         </activity>
         <activity
             android:name=".screens.MainActivity"
+            android:launchMode="singleTask"
```

Comment:
I have known about it, but didnt use correct. If I now right understood this mode will call onNewIntent with new parameter  "int position" in Intent?

Reply:
If you have existing activity all future intents will proceed to OnNewIntent.

You had another option:
1) Start list activity for result
2) In list activity on select of image setResult(intent with image position)
3) Finish list activity
4) Get intent with position in main activity